lab4/context.tex

```diff
@@ -8,8 +8,55 @@ \subsection{Tech stack overview}
 
 \subsection{System change overview}
 
-TODO: quiz change overview (a paragraph that summarizes changes listed in lab 3). To make change harder, instead of simple quiz taking session, write about quizzes with multiple people (basically kahoot: user can create a new room, others get a code, enter it and will be able to compete and see results and what is their place (Details up to your imagination))
+Following the latest addition to the EduPal system of an integrated quiz taking feature, which allows regular users to view a list of quizzes on a certain topic and take a quiz with shuffled questions and receive instant feedback upon answering a question. Topic owners, on the other hand, are able to do everything a regular user is, plus create, edit or delete quizzes. A new feature is being added to the system - multiple participant quizzes. This feature will allow users to create a quiz room, receive a randomly generated code, share it with other users, and compete in real-time to see who can answer the most questions correctly. The system will display the results of the quiz to all participants and show their place in the quiz. Users may earn win streaks and achievements when participating in quizzes.
```

Comment:
Don't mention single user quizzes, we focus solely on quiz rooms (as the only way to take the quiz) - less stuff to model in later viewpoints.